x-pack/plugins/apm/server/routes/alerts/rule_types/anomaly/register_anomaly_rule_type.ts

```diff
@@ -174,7 +174,6 @@ export function registerAnomalyRuleType({
                     range: {
                       timestamp: {
                         gte: dateStart,
-                        format: 'epoch_millis',
```

Comment:
Are you sure about this change? Don't we pass an epoch timestamp anymore?

Reply:
I checked the other endpoints, and we don't pass it anywhere else, also `dateStart` is typed as string, I had a conversation about it on slack